packages/compass-e2e-tests/tests/logging.test.ts

```diff
@@ -31,8 +29,17 @@ describe('Logging and Telemetry integration', function () {
 
       try {
         await browser.connectWithConnectionString();
-        await browser.shellEval('use test');
-        await browser.shellEval('db.runCommand({ connectionStatus: 1 })');
+
+        if (TEST_MULTIPLE_CONNECTIONS) {
+          // make sure we generate the screen event that the tests expect
+          await browser.navigateToMyQueries();
+        }
```

Comment:
we probably need to update this, right? because in reality the screen event for 'My Queries' won't be emitted or different ones will be emitted upon connection which we will need to reflect here?
Should we ticket this?

Reply:
for the purpose of testing`telemetry.screens()`, this seems fine to me, and actually probably makes the test more resilient since we don't just assume which screen is open by default